video/rooms/display-camera-preview/display-camera-preview.swift

```diff
@@ -2,8 +2,8 @@
 capture video from the front or rear-facing device camera  */
 var camera = TVICameraCapturer()
 
-// Video Tracks (local or remote) can be attached to any UIView 
-@IBOutlet weak var previewView: UIView!
+// TVIVideoView (local or remote) can be added as renderer on TVIVideoTrack
+let previewView = TVIVideoView.init(frame: rect)
```

Comment:
Suggestion:

`// TVIVideoView is a TVIVideoRenderer and can be added to any TVIVideoTrack.`